File: HrioTakeEditorByJava/app/src/main/java/hiro/take/hriotakeeditorbyjava/page_viewer.java
package hiro.take.hriotakeeditorbyjava;

import android.os.Bundle;
import android.webkit.WebView;

import androidx.appcompat.app.AppCompatActivity;

public class page_viewer extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.viewer);
        webviewer();
    }

    private void webviewer() {
        WebView v = findViewById(R.id.viewer);
        v.loadUrl("https://www.google.com");

    }
}

File: TakeHiroEditor-8.1/app/src/main/java/hiro/take/takehiroeditor/PageViewer.kt
package hiro.take.takehiroeditor

import android.Manifest
import android.content.Intent
import android.content.pm.PackageManager
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.widget.*
import androidx.appcompat.app.AlertDialog
import androidx.core.app.ActivityCompat
import androidx.core.content.ContextCompat
import hiro.take.takehiroeditor.R
import java.io.*


class PageViewer : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.pageviewer)
    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {

        super.onCreateOptionsMenu(menu)
        val inflater = menuInflater
        inflater.inflate(R.menu.main, menu)

        return true;
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {

            return true
        }
}